AsyncSwift/Views/EventView.swift

```diff
@@ -5,16 +5,143 @@
 //  Created by Kim Insub on 2022/09/06.
 //
 
+/* TODO: 내용
+ 1. light mode only
+ 2. lock landscape
+ 2. 리스트의 세버론 웨이트 라이트로 적용하기
+ 3. 디바이더 너비 키우기
+ 4. 예정된 행사 테그 구현하기
+ */
+
 import SwiftUI
 
+struct EventModel {
+
+    var event: Event
+    var sessions: [Session]
```

Comment:
서버에서 가져온 값을 그대로 보여주는 것이라면 상수가 적절하지 않을까요 ? :eyes:
```suggestion
    let event: Event
    let sessions: [Session]
```

Reply:
호옥시 JSON Parsing 전 에 화면을 표시하기 위해서 var를 사용하신 걸까요 ?????